config/initializers/rack/attack.rb

```diff
@@ -30,8 +30,8 @@ class Attack
       track_and_return_ip(request.env["HTTP_FASTLY_CLIENT_IP"])
     end
 
-    throttle("message_throttle", limit: 2, period: 1) do |request|
-      if request.path.starts_with?("/messages") && request.post?
+    throttle("message_tag_throttle", limit: 2, period: 1) do |request|
```

Comment:
I wonder if 2 will be a problem with mods who accesses `/t/tag/edit` but we can raise it if it becomes a problem

Reply:
If we have js that can throw more than 2 requests at a time to the server for editing then it might be an issue. But if folks are visiting the edit page themselves I find it highly unlikely they could make more than 2 changes in under a second.